app/jobs/regular/group_smtp_email.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_dependency 'email/sender'
+
+module Jobs
+  class GroupSmtpEmail < ::Jobs::Base
+    sidekiq_options queue: 'critical'
+
+    def execute(args)
+      group = Group.find_by(id: args[:group_id])
+      post = Post.find_by(id: args[:post_id])
+      email = args[:email]
+
+      Rails.logger.debug("[IMAP] Sending email for #{group.name} (#{group.id}) and post #{post.id}.")
```

Comment:
Did we mean to leave this in? Usually debugging information like this behind some kind of flag or setting.

Reply:
I believe that production instances usually have logging level set to `>= warn` so it is not an issue and is super handy in development.